setup.cfg

```diff
@@ -46,6 +46,7 @@ test =
     pytest
     pytest-asyncio
     pytest-mock
+    mock; python_version<'3.8'
```

Comment:
Is it really only needed for python 3.7 or older? Then good news everyone: #70

Reply:
Yes, `mock` is the official backport of `unittest.mock`. For `<3.8` it is necessary since `AsyncMock` was not included back then.